### PR TITLE
refactored donor panel to venue

### PR DESCRIPTION
**Pull request by debonair**
@bausmeier @danfuterman please review

### CODE REVIEW

**Comment by danfuterman** [comment]
@debonair - I think it'd be better to manage the DB refactors in new liquibase changesets, rather than editing the existing mysql scripts and changesets. Otherwise this looks good.

**Comment by debonair** [comment]
@danfuterman @bausmeier I have fixed the build please test

File: src/backingform/validator/DonationBackingFormValidator.java [hunk 1]
```diff
@@ -80,13 +80,13 @@ public void validate(Object obj, Errors errors) {
         validateBleedTimes(donation.getBleedStartTime(), donation.getBleedEndTime(), errors);
     }
 
-    Location donorPanel = form.getDonation().getDonorPanel();
-    if (donorPanel == null) {
-      errors.rejectValue("donation.donorPanel", "donorPanel.empty",
+    Location venue = form.getDonation().getVenue();
+    if (venue == null) {
+      errors.rejectValue("donation.venue", "venue.empty",
         "Donor Panel is required.");
```
Comment (bausmeier) [inline src/backingform/validator/DonationBackingFormValidator.java]: @debonair The messages here still refer to "Donor Panel".

File: src/backingform/validator/DonationBatchBackingFormValidator.java [hunk 2]
```diff
@@ -47,19 +47,19 @@ public void validate(Object obj, Errors errors) {
...
-    Location donorPanel = form.getDonationBatch().getDonorPanel();
-    ArrayList<Long> donorPanelIds = new ArrayList<>();
-    donorPanelIds.add(donorPanel.getId());
-	if (donorPanel.getId() == null) {
-	  errors.rejectValue("donationBatch.donorPanel", "donorPanel.empty",
+    Location venue = form.getDonationBatch().getVenue();
+    ArrayList<Long> venueIds = new ArrayList<>();
+    venueIds.add(venue.getId());
+	if (venue.getId() == null) {
+	  errors.rejectValue("donationBatch.venue", "venue.empty",
```
Comment (bausmeier) [inline src/backingform/validator/DonationBatchBackingFormValidator.java]: @debonair These messages also still refer to "Donor Panel".

**Comment by bausmeier** [comment]
@debonair There are still a few references. Most of them are in the tests. See my search below:
screenshot from 2015-09-28 14 59 53

**Comment by bausmeier** [comment]
@debonair This looks fine to me. I can't find any more references and everything seems to still work like it did before.

**Comment by debonair** [comment]
@danfuterman have you had a chance to do a review of this?

**Comment by danfuterman** [comment]
Closes #365 (@debonair - please remember to add a reference to the issue that the PR relates to).

@bausmeier, @timler - please take note of these refactor changes, there might be an impact on your current issues (reporting and donor merging).

**Pull request merged**